### DISCUSSION TITLE
koneu/natend repo, dependency of google/nftables(dependency of opensnitch) has been deleted

### DISCUSSION THREAD

**Discussion by CIAvash** [evilsocket/opensnitch#623] [Development]
So dependency koneu/natend has been removed from dependencies of google/nftables on Feb 10.

google/nftables@edf9fe8cd04f49c758262e8b11654d8cb9ec53ed

**Comment by gustavo-iniguez-goya** [comment]
thank you @CIAvash , fixed :+1:
